src/api/Versions.js

```diff
@@ -243,7 +243,6 @@ class Versions extends OffsetBasedAPI {
             currentVersion.version_restored = restoredVersion.version_number;
         }
 
-        // Wrap the current version as a FileVersions object to make merging with the feed easier
```

Comment:
Is this no longer applicable?

Reply:
not really. Now we're merging a version into an existing list instead of arbitrarily wrapping one `BoxItemVersion` as a `FileVersions`. I updated the doc string to be more informative.